metadata-ingestion/src/datahub/ingestion/source/powerbi/m_query/resolver.py

```diff
@@ -215,7 +231,7 @@ def resolve_to_data_platform_table_list(
         ctx: PipelineContext,
         config: PowerBiDashboardSourceConfig,
         platform_instance_resolver: AbstractDataPlatformInstanceResolver,
-    ) -> List[DataPlatformTable]:
+    ) -> List[Lineage]:
```

Comment:
in some places, we replaced List[DataPlatformTable] with `Lineage`, and in some places we replaced it with `List[Lineage]`

given that Lineage has a `upstreams: List[DataPlatformTable]` field, we should probably be returning `Lineage` instead of `List[Lineage]` here and elsewhere